core/src/main/java/org/mule/runtime/core/internal/exception/ErrorHandler.java

```diff
@@ -201,6 +206,9 @@ private void addDefaultErrorHandlerIfRequired() throws InitialisationException {
     initialiseIfNeeded(acceptsAllOnErrorPropagate, muleContext);
 
     if (this.getLocation() != null && shouldAddLocationToDefaultErrorHandler()) {
+      String location = this.getLocation().getLocation();
+      String containerLocation =
+          inDefaultErrorHandler ? location : location.substring(0, location.length() - ERROR_HANDLER.length() - 1);
```

Comment:
use the items from the location instead of processing the string representation of the location

Reply:
this is not used anymore, it was added in the rebase by mistake